### PR TITLE
fix(kre-py): fix hardcoded value for compressing messages

### PR DESCRIPTION
**Pull request by JYisus**
## Description

Compression protocols will now take the maximum permitted payload size from NATS configuration.

## Motivation and Context

Users wanted to send payloads larger than 1MB of size.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [X] I have created tests for my code changes, and the tests are passing.
- [X] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: e2e/test_assets/docker-compose.yml [hunk 1]
```diff
@@ -20,13 +20,20 @@ services:
       "INFLUXDB_DB": "runtime1"
       "INFLUXDB_META_DIR": "/var/lib/influxdb/meta2"
 
+    logging:
```
Comment (alexrodfe) [inline e2e/test_assets/docker-compose.yml:23]: all logging should be removed

File: e2e/test_assets/nodeC-vol/src/node/main.go [hunk 2]
```diff
@@ -12,6 +12,16 @@ import (
 	"github.com/konstellation-io/kre-runners/kre-go/v4"
 )
 
+const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+func randStringBytes(n int) string {
```
Comment (alexrodfe) [inline e2e/test_assets/nodeC-vol/src/node/main.go:17]: no longer needed

**Review by dfinteligenz — APPROVED** [review]

**Pull request merged**